### PR TITLE
Adding rate limit processor doc to processors list

### PR DESCRIPTION
**Pull request by ycombinator**
## What does this PR do?

Adds the `rate_limit` processor document to the list of processors document. This was missed in #22883 and thanks to @jsoriano for pointing it out.

## Why is it important?

So the `rate_limit` processor's documentation is discoverable to users.

## Checklist

- [ ] ~My code follows the style guidelines of this project~
- [ ] ~I have commented my code, particularly in hard-to-understand areas~
- [x] I have made corresponding changes to the documentation
- [ ] ~I have made corresponding change to the default configuration files~
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by ycombinator** [comment]
Hey @dedemorton, I'm trying to run `make docs` on this PR from the root folder of my `beats` repo checkout. I'm getting the following error.

```
development/github/beats ‹lb-processor-rate-limit-doc*› » make docs
Cloning into '/Users/shaunak/development/github/beats/build/docs'...
remote: Enumerating objects: 548, done.
remote: Counting objects: 100% (548/548), done.
remote: Compressing objects: 100% (476/476), done.
remote: Total 548 (delta 54), reused 301 (delta 20), pack-reused 0
Receiving objects: 100% (548/548), 41.51 MiB | 11.69 MiB/s, done.
Resolving deltas: 100% (54/54), done.
Building docs for libbeat...
Index document: /Users/shaunak/go/src/github.com/elastic/beats/libbeat/docs/index.asciidoc
Index path: docs
Destination directory: /Users/shaunak/development/github/beats/build/html_docs/libbeat/docs
Can't mount two repos with the same name [beats]
make[1]: *** [docs] Error 1
make: *** [docs] Error 1
```

File: libbeat/docs/processors-list.asciidoc [hunk 1]
```diff
@@ -86,6 +86,9 @@ endif::[]
 ifndef::no_include_fields_processor[]
 * <<include-fields,`include_fields`>>
 endif::[]
+ifndef::no_include_rate_limit_processor[]
+* <<rate-limit,`rate_limit`>>
```
Comment (dedemorton) [inline libbeat/docs/processors-list.asciidoc:90]: This line just adds a link. To fix the doc build error, you also need to add the include in the correct location under `//# tag::processors-include[]`

Comment (dedemorton) [reply libbeat/docs/processors-list.asciidoc:90]: Specifically, you want to add this to the correct location (alphabetically) in the list of includes:

```
ifndef::no_include_rate_limit_processor[]
include::{libbeat-processors-dir}/ratelimit/docs/rate_limit.asciidoc[]
endif::[]
```

**Comment by dedemorton** [comment]
Regarding:
>I'm trying to run make docs on this PR from the root folder of my beats repo checkout. I'm getting the following error.

I get a totally different error when I run `make docs` from the `beats` directory:

```
mage docs
Error parsing magefiles: error running "go list -f {{.Dir}}||{{.Name}} github.com/elastic/beats/v7/dev-tools/mage/target/build": exit status 1
can't load package: package github.com/elastic/beats/v7/dev-tools/mage/target/build: cannot find package "github.com/elastic/beats/v7/dev-tools/mage/target/build" in any of:
	/usr/local/go/src/github.com/elastic/beats/v7/dev-tools/mage/target/build (from $GOROOT)
	/Users/dedemorton/go/src/github.com/elastic/beats/v7/dev-tools/mage/target/build (from $GOPATH)
make[1]: *** [docs] Error 1
```

However, when I change to a specific directory, it works. 

I'm not sure what has changed, but I can't help you. This sounds like a bug in the scripts. Maybe ask in the beats slack channel if you haven't already?

File: libbeat/docs/processors-list.asciidoc (hunk 1 above)
Comment (ycombinator) [reply libbeat/docs/processors-list.asciidoc:90]: Thanks, fixed in a31b5ac53.

**Review by jsoriano — APPROVED** [review]
Preview in docs build looks good :+1:

**Pull request merged**